### PR TITLE
added policies

### PR DESCRIPTION
**Pull request by ch-stark**
added three policies and tested them to work. It needs to be highlighted (and tried to do that) that they need to be modified and adjusted to env to work properly

### CODE REVIEW

File: community/CM-Configuration-Management/policy-cluster-dns-sample.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-config-dns
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-cluster-dns-sample.yaml]: change standards/categrories/controls to NIST 800-53

File: community/CM-Configuration-Management/policy-cluster-network-sample.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-config-network
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-cluster-network-sample.yaml]: change standards/categrories/controls to NIST 800-53

File: community/CM-Configuration-Management/policy-cluster-proxy-sample.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-config-proxy
+  annotations:
+    policy.open-cluster-management.io/standards: NIST-CSF
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-cluster-proxy-sample.yaml]: change standards/categrories/controls to NIST 800-53

File: community/CM-Configuration-Management/policy-cluster-dns-sample.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-config-dns
+  annotations:
+    policy.open-cluster-management.io/standards: NIST 800-53
+    policy.open-cluster-management.io/categories: PR.IP Information Protection Processes and Procedures
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-cluster-dns-sample.yaml]: same issue as https://github.com/open-cluster-management/policy-collection/pull/23/files

**Review by ycao56 — APPROVED** [review]

**Comment by ch-stark** [comment]
I need to fix the readme

**Comment by ch-stark** [comment]
from my point of view should be resolved but happy for any review

**Review by dockerymick — APPROVED** [review]
I made a couple of small updates. The changes look good

**Pull request merged**